src/components/structures/TimelinePanel.js

```diff
@@ -496,10 +496,15 @@ var TimelinePanel = React.createClass({
         if (!MatrixClientPeg.get()) return;
 
         var currentReadUpToEventId = this._getCurrentReadReceipt(true);
-        var currentReadUpToEventIndex = this._indexForEventId(currentReadUpToEventId);
+        // Take the minimum of both in case they somehow get out of sync where the RM is
```

Comment:
is this in any way related to the PR at hand? I mean, it's probably fine, but I don't understand what problem you are trying to solve, and it doesn't look like it's the one in the PR summary